infra/chromadb_compaction/lambdas/utils/metrics.py

```
@@ -0,0 +1,373 @@
+                "Timestamp": int(
+                    time.time() * 1000
+                ),  # EMF expects milliseconds
+                "CloudWatchMetrics": [
+                    {
+                        "Namespace": self.namespace,
+                        "Dimensions": (
+                            [list(dimensions.keys())] if dimensions else [[]]
+                        ),
+                        "Metrics": [
+                            {"Name": name, "Unit": "Count"}
+                            for name in metrics.keys()
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)